dpppt-backend-sdk/dpppt-backend-sdk-ws/src/test/java/org/dpppt/backend/sdk/ws/controller/GaenV2ControllerTest.java

```diff
@@ -97,6 +97,63 @@ public void testNoTokenFails() throws Exception {
     assertTrue(authenticateError.contains("Bearer"));
   }
 
+  private void testJWTFakeNoKeysInserted(boolean useV2Upload) throws Exception {
```

Comment:
Wouldn't it make more sense to replace the `boolean` with a `int` here? We'll probably not be going above v2, but still...

```suggestion
  private void testJWTFakeNoKeysInserted(int APIversion) throws Exception {
```